evals/data/eval-gold-responses.json

```diff
@@ -0,0 +1,25 @@
+{
+  "issueResponses": [
+    {
+      "scenario": "manifest.name mismatch ",
+      "issue": {
+        "body": "Manifests need to be updated so the name matches the intended name, which is the name of the repo it lives in.\n\nAny mismatch in manifest.name and the plugin repo, and we will not be able to install those plugins. The config will look like this:\n\nThis is because the worker URL contains the repo name, and we use that to match against manifest.name.",
+        "number": 27,
+        "html_url": "https://github.com/ubiquity-os/ubiquity-os-plugin-installer/issues/27/",
+        "question": "@ubosshivaditya could you please provide a summary of the issue ?"
```

Comment:
Will it be a problem to leave in `@ubosshivaditya`?

Also this seems like a random example can you explain the context of this file further?

Reply:
I guess it's "gold standard" I just messed up the terminology when I called it gold star.

Reply:
Hard coding seems questionable for developers but generally yes I agree that it's easier to deal with vs secrets. @gentlementlegen  please decide